landoui/forms.py

```diff
@@ -90,3 +93,83 @@ class UserSettingsForm(FlaskForm):
         ]
     )
     reset_phab_api_token = BooleanField('Delete', default="")
+
+
+class YesNoField(RadioField):
+    """
+    A simple boolean field displayed as a list of radio buttons
+    Will output a bool value (or None on unknown)
+    """
+
+    def post_validate(self, *args, **kwargs):
+        values = {'yes': True, 'no': False, 'unknown': None}
+        self.data = values.get(self.data)
+
+
+class ApprovalRequestForm(FlaskForm):
+    """Form to create a new approval request"""
```

Comment:
Likewise, this docstring should be uplift-specific.  "approval request" doesn't specify which of the different kinds of lando approval request this class works with.